src/components/charts/miss-distance-chart/getMissDistanceChartDatasets.ts

```diff
@@ -0,0 +1,113 @@
+import type { ChartData } from 'chart.js';
+
+import { dayjs, eachDayOfInterval } from '@/libs/Dayjs';
+
+import { chartPalette } from '../utils/theme';
```

Comment:
maybe move it to place where you have chart base components? we will use it only for charts so there is no sense to have it in utils

Reply:
utils removed